### PR TITLE
Fix error copying postgresql credentials

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

There was an error in copying PostgreSQL credentials being formated as MySQL credentials for the database, this is being fixed by this PR

#### Ticket

https://trello.com/c/QPAV5oFA

#### Screenshots
Screenshot from 2021-07-01 14-06-22

### CODE REVIEW

File: src/components/DBSettingsPage/index.jsx [hunk 1]
```diff
@@ -170,6 +171,15 @@ class DBSettingsPage extends React.Component {
     this.setState({ uriChecked: true });
   }
 
+  uriPOnClick() {
+    const { databaseID } = this.props.match.params;
+    const dbInfo = this.getDatabaseInfo(databaseID);
+    navigator.clipboard.writeText(
+      `${`psql -h ${dbInfo.host} -p ${dbInfo.port} -d ${dbInfo.name} -u ${dbInfo.user}`}`
```
Comment (MutegekiHenry) [inline src/components/DBSettingsPage/index.jsx:178]: @neelxie still missing a something the `-u` is supposed to be capital `-U` and the flag that prompts for a password `-W` is missing

**Review by MutegekiHenry — CHANGES_REQUESTED** [review]
Just a few more changes

File: src/components/DBSettingsPage/index.jsx (hunk 1 above)
Comment (MutegekiHenry) [reply src/components/DBSettingsPage/index.jsx:178]: Unless this PR is strictly for copying issue and another for the actual command awaits

Comment (neelxie) [reply src/components/DBSettingsPage/index.jsx]: @MutegekiHenry thank you, I have fixed that.

File: src/components/DBSettingsPage/index.jsx [hunk 2]
```diff
@@ -328,10 +338,10 @@ class DBSettingsPage extends React.Component {
                     </div>
                   </div>
                   <div className="DBInfoBottom">
-                    <div className="DBAccessInfo">{`psql -h ${dbInfo.host} -p ${dbInfo.port} -d ${dbInfo.name} -u ${dbInfo.user}`}</div>
+                    <div className="DBAccessInfo">{`psql -h ${dbInfo.host} -p ${dbInfo.port} -d ${dbInfo.name} -u ${dbInfo.user} -W`}</div>
```
Comment (MutegekiHenry) [inline src/components/DBSettingsPage/index.jsx:341]: @neelxie forgot the `-U` casing

File: src/components/DBSettingsPage/index.jsx (hunk 1 above)
Comment (MutegekiHenry) [reply src/components/DBSettingsPage/index.jsx]: Awesome!

File: src/components/DBSettingsPage/index.jsx (hunk 2 above)
Comment (neelxie) [reply src/components/DBSettingsPage/index.jsx:341]: @MutegekiHenry this has been fixed

**Review by MutegekiHenry — APPROVED** [review]
Looks good.

**Review by kawerewagaba — APPROVED** [review]
lgtm

**Pull request merged**